src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
     </div>
   );
 }
```

Comment:
- [ ] App.js should have a `class-based` component as present in **src/compnenets/calculator.js**.

Reply:
Thanks for your time @Arslan. I would love to implement those changes. However, I don't understand what you mean. Could you please guide me to your explanation so I can clarify my doubts?

#### 1st Doubt 
The snippet code that you highlight there doesn't belong to my app or at least in my browser the rendered app architecture looks like follows :
```
<div class="App"><div class="calculator-body"><div class="body-result"><h5>0</h5></div><div class="body-buttons"><div class="buttons-row-1"><button type="button" class="col-1 ">AC</button><button type="button" class="col-2 ">+/-</button><button type="button" class="col-3 ">%</button><button type="button" class="col-4 orange">/</button></div><div class="buttons-row-2"><button type="button" class="col-1 ">7</button><button type="button" class="col-2 ">8</button><button type="button" class="col-3 ">9</button><button type="button" class="col-4 orange">x</button></div><div class="buttons-row-3"><button type="button" class="col-1 ">4</button><button type="button" class="col-2 ">5</button><button type="button" class="col-3 ">6</button><button type="button" class="col-4 orange">-</button></div><div class="buttons-row-4"><button type="button" class="col-1 ">1</button><button 
```

Reply:
What I meant is please change app.js code to class-based.

Reply:
:heavy_check_mark:  Understood

Reply:
:heavy_check_mark: 1st Request : " What I meant is please change app.js code to class-based."